L/LibCURL/build_tarballs.jl

```diff
@@ -76,8 +76,8 @@ dependencies = [
     Dependency("nghttp2_jll"),
     # Note that while we unconditionally list MbedTLS as a dependency,
     # we default to schannel/SecureTransport on Windows/MacOS.
-    Dependency("MbedTLS_jll", v"2.24.0"),
+    Dependency("MbedTLS_jll"; compat="~2.28.0"),
```

Comment:
```suggestion
    Dependency("MbedTLS_jll"; compat="~2.28.0"),
    Dependency("LibSSH2_jll"),
```

Just reading the last remaining error, are you missing the following (above)?

Reply:
Interesting.  Why is this only needed on one platform?

Reply:
No https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=15920&view=logs&j=a6122f81-e5a2-5211-451a-3f540ebd06e0&t=ff55cc42-c3ce-53e5-62b6-7d1c7f90b9e0&l=3499